src/KingSystem/ActorSystem/actActor.h

```diff
@@ -285,6 +285,9 @@ class Actor : public BaseProc, public ActorMessageTransceiver::IHandler {
     virtual void m146();
     virtual void m147();
 
+    auto* get68f() { return &_68f; }
```

Comment:
I think it would make more sense for this to return a reference. Or even a bool?